lib/tasks/elasticsearch.rake

```diff
@@ -0,0 +1,9 @@
+namespace :elasticsearch do
+  desc 'Delete the "resource" index'
+  task :delete_resource_index do
+    client = Elasticsearch::Client.new(
```

Comment:
@adarsh Good question.

The only place I see it being instantiated is in `config/initializers/elasticsearch.rb`, where we assign it to `Elasticsearch::Model.client`. 

It feels incorrect to access that in order to access ES. What do you think?